editor/plugins/path_editor_plugin.h

```diff
@@ -78,6 +84,7 @@ class PathEditorPlugin : public EditorPlugin {
 	static PathEditorPlugin *singleton;
 	Ref<SpatialMaterial> path_material;
 	Ref<SpatialMaterial> path_thin_material;
+	CheckBox *mirror_handles;
```

Comment:
You aren't using this anywhere else.

Reply:
oops, a left over relic, I'll remove that now. Thanks for the git advice too, I'll try it now